### ISSUE TITLE
Accessibility Abide: Major violation (Urgent Fix required)

### ISSUE THREAD

**Issue by wsansewjs** [zurb/foundation#6372]
NO, NO, and NO!

Please do not call Abide accessible when it is not even meeting the HTML5 standards, even WCAG 2.0.

Please refer to the docs at http://foundation.zurb.com/docs/components/abide.html and scroll down to the Accessibility section.

Here is the example code that is **INCORRECT**.

```html
<form data-abide>
  <div class="name-field">
    <label>Your name <small>required</small>
      <input type="text" required pattern="[a-zA-Z]+">
    </label>
    <small class="error" id="nameError">Name is required and must be a string.</small>
  </div>
</form>
```

**HERE IS THE CORRECT VERSION:**

```html
<form data-abide>
  <div class="name-field">
    <label for="input-example">Your name <small>required</small></label>
    <input id="input-example" type="text" required pattern="[a-zA-Z]+">
    <small class="error">Name is required and must be a string.</small>
  </div>
</form>
```

How can someone forget the BASIC requirements for form elements such as the ID and FOR attributes? Shame! You do NOT put the input, checkboxes, radio buttons, textarea, etc. inside the LABEL tag for Accessibility, period!

I will be raising other major accessibility issues with Abide in the near future.

**Comment by rafibomb** [comment]
Thanks for the feedback! We're not claiming to be accessibility experts (yet) but for the next version it is a huge focus. We appreciate you pointing this out. As we develop the next version, can we get your input on accessibility of components?

Made the update here: https://github.com/zurb/foundation/commit/b094fd56919e39d494c80d144796dde96df68cce
